### ISSUE TITLE
part 3d validation on update

### ISSUE THREAD

**Issue by Data5tream** [fullstackopen-2019/fullstackopen-2019.github.io#615]
Mongoose doesn't run validation on default when updating objects and needs to be run with the ```runValidators: true``` option. This should be mentioned in exercise 3.19.

**Comment by Kaltsoon** [comment]
Yes, that's a good point. Perhaps we should mention it when introducing the `findByIdAndUpdate` method in [part 3c](https://fullstackopen.com/en/part3/saving_data_to_mongo_db)
